bin/download-logs

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require_relative '../lib/maze'
+require_relative '../lib/maze/browser_stack_utils'
+require_relative '../lib/maze/logger'
+require 'net/http'
+require 'json'
+require 'uri'
+require 'fileutils'
+
+class DownloadLogEntry
+  def start(args)
+    # TODO change this to --file and --build-id
```

Comment:
For this PR can we print a simple usage message if either no args of `--help` is given?